locale/es.yml

```diff
@@ -1248,113 +1248,113 @@ __TRAIT__: ""
 # |  LANGUAGE: PYTHON  |
 # ----------------------
 # << Keywords >>
-  False: ""
-  None: ""
-  True: ""
-  and: ""
-  as: ""
-  assert: ""
-  async: ""
-  await: ""
-  break: ""
-  class: ""
-  continue: ""
-  def: ""
-  del: ""
-  elif: ""
-  else: ""
-  except: ""
-  finally: ""
-  for: ""
-  from: ""
-  global:  ""
-  if: ""
-  import: ""
-  in: ""
-  is: ""
-  lambda: ""
-  nonlocal: ""
-  not: ""
-  or: ""
-  pass: ""
-  raise: ""
-  return: ""
-  try: ""
-  while: ""
-  with: ""
-  yield: ""
+  False: "Falso" # ✅✅✅✅✅✅✅
+  None: "Ninguno" # ✅✅✅✅✅✅✅
+  True: "Verdadero" # ✅✅✅✅✅✅✅
+  and: "Y" # ✅✅✅✅✅✅✅
+  as: "como" # ✅✅✅✅✅✅
+  assert: "afirmar" # ✅❓ What about "asegurar"? ✅ I agree with "afirmar" ✅ ✅ I also agree with "afirmar"✅✅ I agree with "afirmar"
```

Comment:
I believe that keeping afirmar would be the best decision as it seems to translate pretty well from assert. I also notice that a lot of other reviewers thought the same, and so maybe afirmar should stay.